fix: prevent user dropdown redirect on Talks page

siddharthg2309 opened the pull request:
Fixes #1568

## Overview
On the Talks page, clicking the logged-in user in the top navigation redirected to Account Settings instead of opening the user dropdown. This behavior was inconsistent with the Tickets page.

## Solution
- Reused the existing `popover.js` implementation from the Tickets page
- Ensured required data attributes and dependencies are present
- Prevented navigation redirect and restored dropdown behavior

## Result
Clicking the user name on the Talks page now correctly opens the user dropdown, consistent with the Tickets page behavior.

ArnavBallinCode commented:
> @Saksham-Sirohi I have reverted the design changes and focused specifically on the core issue: redirection to account settings instead of the dropdown.

can you please share the screen recording for the updated

siddharthg2309 commented:
@ArnavBallinCode 
screen recording for the updated

https://github.com/user-attachments/assets/a1c74ef2-4b1a-44a5-b244-91ea239c7b07

ArnavBallinCode commented on app/eventyay/orga/templates/orga/base.html:
```diff
@@ -143,10 +141,10 @@
                 {% block navbar_right %}{% endblock navbar_right %}
                 <li class="nav-item d-none d-md-block">
-                    <a class="nav-link active" href="{% url "eventyay_common:account.general" %}">
+                    <span class="nav-link active" style="cursor: pointer; color: var(--color-offwhite, white);" onclick="setTimeout(() => document.querySelector('#global-nav-menu button')?.click(), 0)">
```
This inline onclick handler violates CSP and creates a maintenance nightmare. The setTimeout(0) suggests a race condition - the Vue component might not be mounted till then. 


eg: `<span id="user-dropdown-trigger" class="nav-link active user-dropdown-trigger">`

ArnavBallinCode reviewed (changes requested):
These changes reinvent the wheel. The solution is to reuse the existing popover implementation from the Tickets page, not create a brand new Vue 3 component with minified bundles.

siddharthg2309 commented:
@ArnavBallinCode Implemented as suggested:
- Reused popover.js from Tickets page
- Added required dependencies (jQuery, Bootstrap, utils.js)
- Added data attributes for popover configuration
- Added _popover.css extracted popover styles to avoid main.scss conflicts

Screenshot for the updated
<img width="1469" height="956" alt="image" src="https://github.com/user-attachments/assets/9cb148a0-9807-4b55-b6ff-a3d2bc899b9e" />

ArnavBallinCode reviewed (changes requested):
Remove issue number from the title

ArnavBallinCode commented on app/eventyay/orga/templates/orga/base.html:
```diff
@@ -143,12 +146,11 @@
                 {% block navbar_right %}{% endblock navbar_right %}
                 <li class="nav-item d-none d-md-block">
-                    <a class="nav-link active" href="{% url "eventyay_common:account.general" %}">
+                    <a class="nav-link active" data-toggle="popover-profile" style="cursor: pointer;">
```
`<a class="nav-link" data-toggle="popover-profile" style="cursor: pointer;">
`

The 'active' class should not be permanently applied. It makes the link always appear selected, which is incorrect UX. Remove the 'active' class.

ArnavBallinCode commented on app/eventyay/static/orga/css/_popover.css:
```diff
@@ -0,0 +1,153 @@
+    z-index: 1060;
+    background: white;
+    position: absolute !important;
+    top: 50px !important;
```
`position: absolute;`

popover plugin automatically calculates the correct position. Hardcoding `top: 50px` breaks this functionality and causes misalignment on different screen sizes.

mariobehling reviewed (changes requested):
Please consider https://github.com/fossasia/eventyay/issues/1367. This issue should actually be solved before aligning tickets and talks. The dropdown menu in tickets does not show the icons as they are white. This bug appeared recently and it should be solved first. https://github.com/fossasia/eventyay/issues/1367 is a blocker for the issue here.

mariobehling commented:
Thank you for your contribution and for putting in the effort on this PR.

After reviewing the maintainers’ feedback and the existing work in the repository, there are several reasons we cannot merge this change:

* The scope of this PR does not align with the current responsibilities and roadmap of the `eventyay` repository. This area of functionality is not considered in scope here as confirmed by a core maintainer.
* Portions of what this PR attempts to address are already being covered or reworked elsewhere, which means merging this would introduce duplication and potential conflicts.
